src/npm/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+if (process.env.NODE_ENV === 'production') {
+    module.exports = require('./recoil.production.js');
+} else {
+    module.exports = require('./recoil.development.js');
+}
```

Comment:
Is this a common pattern to do that? I’ve never seen that before

Reply:
Not sure how common it is, I copied this straight from React and it works well enough for this use case.

https://github.com/facebook/react/blob/master/packages/react/npm/index.js

Reply:
Thanks, if React is doing it, it's probably safe :)